tools-ui/src/test/java/com/hedera/hashgraph/client/ui/CreatePaneControllerTest.java

```diff
@@ -950,6 +956,122 @@ public void createUpdateAccount_Test() throws HederaClientException {
 		assertTrue(comment.has("Timestamp"));
 	}
 
+	@Test
+	public void createLargeBinaryUpdate_test() throws HederaClientException, IOException {
```

Comment:
nit: This test is long and hard to follow. Should consider breaking it up into smaller helper functions, especially to show clearly the test setup (given, when) and the expectation (then). Also, many of the var here could also be `final`.

Reply:
Thanks I will break it up